Geometry/GlobalTrackingGeometryBuilder/plugins/GlobalTrackingGeometryBuilder.cc

```diff
@@ -33,13 +35,13 @@ GlobalTrackingGeometry* GlobalTrackingGeometryBuilder::build(const TrackerGeomet
     std::vector<const TrackingGeometry*> tkGeometries;
     
     tkGeometries.emplace_back(tk);
+    tkGeometries.emplace_back(mtd);
```

Comment:
I changed the enum in the appropriate way, and all corresponding tests past.

As it turns out a lot of copy/paste code is needed if you want to have separate alignment records for each subdetector which must be aligned.

Reply:
Are there any alignment records (e.g. DB payloads) which may depend on the order of the geometries? Will we need to back port the enum and update the GTs?

I think, if a code copy is necessary, it's worth spending a little bit more time to make it better then the original.

Reply:
Nope, the global tracking geometry itself is never stored in the database (as far as I can tell) and is only ever derived from the pre-computed, since all it does is store pointers over to other geometries.

Reply:
** and the alignment records are associated with the specific geometry implementation rather than the global geometry (which makes sense)